readme.md

```diff
@@ -130,6 +130,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [Terminology](https://github.com/billiob/terminology) - The best terminal emulator based on the Enlightenment Foundation Libraries.
 * [Terminus](https://github.com/Eugeny/terminus) - Cross-platform terminal for a more modern age, based on web technologies.
 * [Termite](https://github.com/thestinger/termite/) - Keyboard-centric terminal, aimed at use within a tiling window manager.
+* [Termius](https://www.termius.com/) - Termius is the SSH client that works on Desktop and Mobile.
```

Comment:
Please change `Termius is the SSH client that works on Desktop and Mobile.` to `Desktop and Mobile SSH client`